stdlib/Project.toml

```diff
@@ -0,0 +1,61 @@
+[deps]
```

Comment:
I think this is absolutely the right step. Thoughts for the future:

Could we get away with the current special way of managing versions in `stdlib` and instead use compat bounds? But that might require Julia to build Julia something that we have so far avoided. But we could totally use Julia 1.11 to build Julia 1.12

Should we have compat bounds here?